.github/workflows/release-master.yml

```diff
@@ -38,21 +38,59 @@ jobs:
         env:
           DOCKER_DRIVER: overlay
           DOCKER_HOST: tcp://localhost:2375
+    permissions:
+      contents: read
+      packages: write
+      id-token: write
     steps:
     - name: Install Tools
-      run: apk add --update make git jq rsync curl
+      run: apk add --update make git jq rsync curl bash coreutils
```

Comment:
just so understand, what's the need for adding `bash` and `coreutils` here?

Reply:
It's installing `bash` because the `cosign-installer` action requires it. 

The version of `base64` in the container image doesn't support the `-w` flag which the `slsa-provenance-action` requires, hence installing `coreutils` to get a more up-to-date version of `base64`.

I'll make this clear in a comment.